Polyhedron/demo/Polyhedron/Scene_points_with_normal_item.cpp

```diff
@@ -467,9 +467,28 @@ void Scene_points_with_normal_item::resetSelection()
 void Scene_points_with_normal_item::selectDuplicates()
 {
   std::set<Kernel::Point_3> unique_points;
-  for (Point_set::iterator ptit = d->m_points->begin(); ptit!= d->m_points->end();++ptit )
+  std::vector<Point_set::Index> unselected, selected;
+  for (Point_set::iterator ptit = d->m_points->begin(); ptit!= d->m_points->end(); ++ ptit)
     if ( !unique_points.insert(d->m_points->point(*ptit)).second)
-      d->m_points->select(ptit);
+      selected.push_back (*ptit);
+    else
+      unselected.push_back (*ptit);
+  
+  for (std::size_t i = 0; i < unselected.size(); ++ i)
+    *(d->m_points->begin() + i) = unselected[i];
+  for (std::size_t i = 0; i < selected.size(); ++ i)
+    *(d->m_points->begin() + (unselected.size() + i)) = selected[i];
+
+  if (selected.empty ())
+    {
+      d->m_points->unselect_all();
+    }
+  else
+    {
+      d->m_points->set_first_selected
+        (d->m_points->begin() + unselected.size());
+    } 
```

Comment:
It seems you use a different coding style from the initial author (and probably different from most CGAL developers), where both the `{` and its content are indented.

1. Is that on purpose? In my opinion that coding style wastes to many horizontal spaces.
2. Anyway, where you modify an existing file that was not created by you, you should try to mimic the existing coding style.